util/predictoor/models.py

```diff
@@ -0,0 +1,47 @@
+from enforce_typing import enforce_types
+from util.constants import MIN_PREDICTIONS
+
+
+class Prediction:
+    @enforce_types
+    def __init__(self, block: int, payout: float, contract_addr: str):
+        self.block = block
+        self.payout = payout
+        self.contract_addr = contract_addr
+
+    @property
+    def correct(self) -> bool:
```

Comment:
I would suggest `is_correct` to emphasize that we are not correcting something, we are checking if something is correct